### DISCUSSION TITLE
Suggestions in reducing the number of migration files

### DISCUSSION THREAD

**Discussion by lilingxi01** [prisma/prisma#24101] [Q&A]
### Question

When developing the database, we run `prisma migrate dev --name a_sample_name` every time we want to make some modifications. Sometimes we can have 20+ modifications on a table as data complexity grows. This ends up getting us a bunch of folders in `/prisma/migrations`. Any suggestions in avoiding having too many migration folders? Any method to combine multiple dev migrations into one before deploying to prod? Or is this an optimal approach?

Context: we use `prisma migrate` instead of `prisma db push` because we use CI for automatically deploying schema to production database.

Thank you!

**Comment by ludralph** [comment]
Hi @lilingxi01 👋 

You can squash multiple migration files into a single migration. This can be useful in a scenario like yours where you have many small and frequent changes to the database schema during development. Squashing migrations can help avoid unnecessary intermediate steps in the final migration history that will be pushed. Kindly look at this section of the docs for [migrating cleanly from a development environment](https://www.prisma.io/docs/orm/prisma-migrate/workflows/squashing-migrations#migrating-cleanly-from-a-development-environment).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
